indolaw-nextjs/components/TableOfContents.tsx

```diff
@@ -22,15 +27,33 @@ export default function TableOfContents(props: { law: Complex }): JSX.Element {
     />
   ));
 
+  const darkModeButton = (<><style>{
+      `button {
+        width: 25%;
+        height: 3vh;
+        margin-bottom: 10px;
+        background-color: ${(isDarkMode ? colors : darkColors).trayBackground};
+        border-radius: 7px;
+        border: 0;
+        color: ${(isDarkMode ? colors : darkColors).dark.text};
+        font-family: ${fonts.sans};
+        ${isMobile ? "float: right;" : "margin-left: auto; display: block;"}
+      }
+      }`}</style><button onClick={setIsDarkMode}>{isDarkMode ? "Light Mode" : "Dark Mode"}</button></>
+  );
+
   if (!isMobile) {
-    return <>{tableOfContents}</>;
+    return <>
+      {darkModeButton}
+      {tableOfContents}
+    </>;
   }
 
   return isExpanded ? (
     <div className="container">
       <style jsx>{`
         .container {
-          background-color: ${colors.background};
+          background-color: ${(isDarkMode ? darkColors : colors).trayBackground};
```

Comment:
@willemchua Same as my comment on LawPage.tsx: let's consolidate the ternary operator in 1 place & assign to a variable we reuse throughout the file